src/components/tableview/TableTable.vue

```diff
@@ -318,6 +323,13 @@ export default {
       }
       return null
     },
+    editorType(dt) {
```

Comment:
maybe its just me, but i think `dataType` would be better then `dt`.

Reply:
Agree, just followed `slimDataType` method style